examples/specs_integration/src/lib.rs

```diff
@@ -0,0 +1,155 @@
+#[macro_use]
+extern crate gdnative as godot;
```

Comment:
For an example, `gdnative` should be imported without an alias, as seen in other examples.